master/buildbot/test/unit/test_stats_service.py

```diff
@@ -154,13 +154,6 @@ def test_influx_service_not_inited(self):
         svc.thd_postStatsValue("test", "test", "test")
         self.assertLogged("Service.*not initialized")
 
-    def test_storage_backend_base(self):
-        # Just initlize and run thd_postStatsValue
-        svc = StatsStorageBase()
```

Comment:
not possible, since it is an abstract class. Use a lower level of class, or create a dummy class inherited from StatsStorageBase directly